packages/circuit-ui/components/PhoneNumberInput/PhoneNumberInput.tsx

```diff
@@ -102,51 +103,90 @@ export interface PhoneNumberInputProps
    */
   locale?: string | string[];
   /**
-   * TODO: Write a description
+   * Country code selector details.
    */
   countryCode: {
     /**
      * Visually hidden label for visually-impaired users.
      */
     label: string;
     /**
-     * TODO: Write a description. See https://en.wikipedia.org/wiki/List_of_country_calling_codes
+     * List of country calling codes to be rendered inside the selector
      */
     options: {
       /**
-       * TODO: Write a description
+       * Country name in two letter ISO 3166 region code format(https://www.iso.org/iso-3166-country-codes.html)
        */
       country: string;
       /**
-       * TODO: Write a description
+       * Country calling codes, see https://en.wikipedia.org/wiki/List_of_country_calling_codes
        */
       code: string;
     }[];
+    /**
+     * Triggers error styles on the component. Important for accessibility.
+     */
+    invalid?: boolean;
     /**
      * Initial country code.
      */
     defaultValue?: string;
     /**
-     * TODO: Write a description.
-     * TODO: The HTML `select` element doesn't support the `readonly` attribute, so this will need to be implemented using CSS and the `aria-readonly` and/or `disabled` attributes. The element must remain perceivable by screen reader users.
+     * Triggers readonly styles on the component.
      */
     readonly?: boolean;
+    /**
+     * Callback when the country code changes.
+     * The callback receives the normalized phone number in the [E.164 format](https://en.wikipedia.org/wiki/E.164),
+     * e.g. `+17024181234`.
+     */
     onChange?: SelectProps['onChange'];
+    /**
+     * The ref to the country code selector HTML DOM element.
+     */
     ref?: ForwardedRef<HTMLSelectElement>;
+    /**
+     * Render prop that should render a left-aligned overlay icon or element.
+     * Receives a className prop.
+     */
+    renderPrefix?: ComponentType<{
+      value?: string | number;
+      className?: string;
+    }>;
   };
   /**
-   * TODO: Write a description
+   * Subscriber number input details.
    */
   subscriberNumber: {
     /**
      * Visually hidden label for visually-impaired users.
      */
     label: string;
+    /**
+     * Placeholder number for the input.
+     */
+    placeholder?: string;
     /**
      * Initial subscriber number.
      */
     defaultValue?: string;
+    /**
+     * Triggers error styles on the component. Important for accessibility.
+     */
+    invalid?: boolean;
+    /**
+     * Triggers readonly styles on the component.
+     */
+    readonly?: boolean;
+    /**
+     * Callback when the subscriber number changes.
+     * The callback receives the normalized phone number in the [E.164 format](https://en.wikipedia.org/wiki/E.164),
+     * e.g. `+17024181234`.
+     */
```

Comment:
Same here. This `onChange` callback is specific to the subscriber number `input` element and receives the change `event` (which only contains the raw subscriber number as the `event.target.value`).

Reply:
Correct, code updated